From 7ceb8f6aa64feefd6036348f3d6282d9745f5bb0 Mon Sep 17 00:00:00 2001
From: Eliad Peller <eliad@wizery.com>
Date: Thu, 5 May 2011 13:39:26 +0300
Subject: [PATCH 44/75] wl12xx: call wl1271_cmd_set_peer_state() in AP mode

After adding a station, call wl1271_cmd_set_peer_state(). This is
required for 11n support.
---
 drivers/net/wireless/wl12xx/main.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/drivers/net/wireless/wl12xx/main.c b/drivers/net/wireless/wl12xx/main.c
index 8445afa..ac3e45b 100644
--- a/drivers/net/wireless/wl12xx/main.c
+++ b/drivers/net/wireless/wl12xx/main.c
@@ -3333,6 +3333,10 @@ static u64 wl1271_op_get_tsf(struct ieee80211_hw *hw)
 	if (ret < 0)
 		goto out_sleep;
 
+	ret = wl1271_cmd_set_peer_state(wl, hlid);
+	if (ret < 0)
+		goto out_sleep;
+
 out_sleep:
 	wl1271_ps_elp_sleep(wl);
 
-- 
1.7.0.4

